NetKAN/RasterPropMonitor-Core.netkan

```diff
@@ -18,11 +18,13 @@
         "crewed"
     ],
     "depends": [
-        { "name": "ModuleManager"            },
-        { "name": "RasterPropMonitor-Legacy" }
+        { "name": "ModuleManager" }
     ],
     "install": [ {
         "find"      : "JSI/RasterPropMonitor",
         "install_to": "GameData/JSI"
+    }, {
+        "find":       "JSI/RPMPodPatches/BasicMFD",
+        "install_to": "GameData/JSI/RPMPodPatches"
```

Comment:
Is it possible to leave a comment here explaining why this is included?

Do you also need to remove this folder from the main RasterPropMonitor package?

Reply:
> Is it possible to leave a comment here explaining why this is included?

Good idea, added.

> Do you also need to remove this folder from the main RasterPropMonitor package?

Yes, that's what the `"filter"` property does, `BasicMFD` will be excluded when installing RasterPropMonitor:

https://github.com/KSP-CKAN/NetKAN/blob/e091ca5cf06c16a5a1fc93e1a3225824aa32567b/NetKAN/RasterPropMonitor.netkan#L46

Reply:
Oops, missed that one. 
Ok, looks good to me.  Ship it!